charts/operators/values.yaml

```diff
@@ -1,2 +1,3 @@
-# empty
 ---
+grafana_subscription_version: 4.8.0
```

Comment:
how do we test these changes in `charts/operators`?

Reply:
Not this PR.